lib/perf_hooks.js

```diff
@@ -205,6 +206,10 @@ class PerformanceNodeTiming extends PerformanceEntry {
     return getMilestoneTimestamp(NODE_PERFORMANCE_MILESTONE_BOOTSTRAP_COMPLETE);
   }
 
+  get idleTime() {
+    return loopIdleTime();
+  }
```

Comment:
please add `idleTime` to `[kInspect]` below otherwise it gets not printed if you type `perf_hooks.performance` in repl.